opentelemetry-sdk/CHANGELOG.md

```diff
@@ -64,6 +64,16 @@
  `LogExporter` trait no longer requires a mutable ref to `self`. If the exporter
  needs to mutate state, it should rely on interior mutability.
  [2764](https://github.com/open-telemetry/opentelemetry-rust/pull/2764)
+- Added Two new methods to the LogRecord struct's public API:
+```rust
+  update_attribute(&Key, &AnyValue) -> Option<AnyValue>
+```
+  - Updates the value of the first occurrence of an attribute with the specified key. 
+  - If the key exists, the old value is returned. If not, the new key-value pair is added, and None is returned.
```

Comment:
the behavior of update-or-insert might call for a different name than `update_attribute`. is `set_attribute` better sounding?